pages/techgenius/tg-installation.md

```diff
@@ -62,7 +62,10 @@ If it does not work, try this:
 ```bash
     sudo apt-get install git
     sudo apt-get install virtualbox
-    sudo apt-get install vagrant
+    curl -o vagrant.deb https://releases.hashicorp.com/vagrant/2.0.1/vagrant_2.0.1_x86_64.deb?_ga=2.145355697.407736321.1516146560-799695961.1515792702
+    sudo apt-get install gdebi*
+    sudo gdebi ./vagrant*.deb
```

Comment:
sudo dpkg -i vagrant.deb

Reply:
you may want to run

sudo apt-get install -fy

to fix dependencies if any is missing

Reply:
so, I could just leave it as is then?

Reply:
sounds good! :)